widget/app_launcher/init.lua

```diff
@@ -770,14 +770,14 @@ local function new(args)
     args.default_app_icon_name = args.default_app_icon_name or nil
     args.default_app_icon_path = args.default_app_icon_path or nil
     args.icon_theme = args.icon_theme or nil
-    args.icons_size = args.icons_size or nil
+    args.icon_size = args.icon_size or nil
```

Comment:
I noticed the arg was called `icons_size` while the code was looking for `icon_size` (on line 589)